omdata/reactivity_utils.py

```diff
@@ -274,3 +288,54 @@ def filter_unique_structures(atoms_list, rmsd_cutoff):
             unique_structures.append(atoms)
             
     return unique_structures
+
+
+def filter_unique_structures(atoms_list):
+    max_energy=-np.inf
+    max_energy_structure=None
+    for ii, atoms in enumerate(atoms_list):
+        if atoms.get_potential_energy() > max_energy:
+            max_energy = atoms.get_potential_energy()
+            max_energy_structure = atoms
+
+    rmsd_cutoff_dict = {0.0: 0.14, 0.18: 0.1, 0.3: 0.07, 0.4: 0.04, 0.6: 0.03}
```

Comment:
```suggestion
    rmsd_cutoff_dict = {0.6: 0.03, 0.4: 0.04, 0.3: 0.07, 0.18: 0.1, 0.0: 0.14}
```
You'll see why later. dict order is guaranteed as of Python 3.7 so I didn't feel like changing to an OrderedDict